Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+pipeline {
+    agent any
+    options {
+        disableConcurrentBuilds()
+    }
+    environment {
+        DOCKER_REPO = "docker.dev.pureport.com"
+        DEB_BUILDER_IMAGE = "pureport/deb-builder"
+        UID = getUid()
+        GID = getGid()
+    }
+    stages {
+        stage('Build') {
+            when {
+                branch "pureport/*"
+            }
+            steps {
+                script {
+                    def workspace = pwd()
+                    sh "git fetch --tags"
+                    sh "docker run --rm -v ${workspace}:/src -v ${workspace}:/output -e USER=${env.UID} -e GROUP=${env.GID} ${env.DOCKER_REPO}/${env.DEB_BUILDER_IMAGE} /src/build-debs.sh"
+                }
+            }
+        }
+        stage('Add to Aptly repo') {
+            when {
+                branch "pureport/stable/*"
+            }
+            steps {
+                script {
+                    echo "Add to Aptly repo..."
+                    sh "aptly repo add frr ."
```

Comment:
Does this need to point at the output directory?

Reply:
Nah. The docker command mounts the current directory as `/output` (`-v ${workspace}:/output`).